### ISSUE TITLE
Kafka Controller isn't resilient to config-kafka-features being deleted/not present

### ISSUE THREAD

**Issue by Cali0707** [knative-extensions/eventing-kafka-broker#3781]
**Describe the bug**
When config-kafka-features is not present as the kafka controller starts, the controller panics and fails to start.

 **Expected behavior**
If the cm is not present, we should create one with default values

 **To Reproduce**
Delete the config map and then restart the kafka controller

 **Knative release version**

 **Additional context**
Add any other context about the problem here such as proposed priority

**Comment by pierDipi** [comment]
No, I don't think we need to create one because that means that the installation is not done correctly and it is ok to panic/start failing

**Comment by pierDipi** [comment]
if we go the route of auto-recovering from wrong installations, if someone deletes the data plane deployments, will we create a new one? 

This is mostly on the operator/admin to ensure the installation is correct
